### DISCUSSION TITLE
Zig-Dex interop

### DISCUSSION THREAD

**Discussion by ted-kord** [google-research/dex-lang#646] [Q&A] [answered]
Just found out about this great budding language and it is very exciting.

I want to be able to use it from [Zig](https://ziglang.org/), which has first class C interop. Can you suggest the best way or strategy to do this?

**Comment by oxinabox** [comment, marked as answer]
Copy the way it is done in Julia or Python

**Reply by ted-kord** [reply]
Thanks  oxinabox and I also just saw [#627](https://github.com/google-research/dex-lang/discussions/627) on the c api .
